foundation/foundation-mda/src/test/java/com/boozallen/aiops/mda/generator/DataRecordsModuleStep.java

```diff
@@ -78,6 +86,21 @@ public void a_pipeline_using_named(String typeName, String implName, String name
         createPipeline(name, typeName, implName);
     }
 
+    @Given("a {string} pipeline using {string} named {string} and a file store named {string}")
+    public void a_pipeline_using_named_and_a_file_store_named(String typeName, String implName, String pipelineName, String fileStoreName) throws IOException {
+        List<String> fileStoreNames = new ArrayList<>();
+        fileStoreNames.add(fileStoreName);
+        createPipeline(pipelineName, typeName, implName, p -> p.setFileStores(createFileStores(fileStoreNames)));
```

Comment:
A: Feels like it would be more straightforward to just have `createPipeline` return the pipeline object. If we're worried about the double-call to `saveToFile`, we could always rename the current method to `createAndSavePipeline` and pull out the creation logic minus `saveToFile` to `createPipeline`.

Reply:
Refactored to split pipeline creation and saving in order to allow separate and clauses for adding file stores to a pipeline.